src/libraries/System.Net.Sockets/tests/FunctionalTests/UdpClientTest.cs

```diff
@@ -264,7 +264,6 @@ public void Ttl_Roundtrips()
             }
         }
 
-        [SkipOnPlatform(TestPlatforms.OSX | TestPlatforms.FreeBSD, "BSD like doesn't have an equivalent of DontFragment")]
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/51392", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
         public void DontFragment_Roundtrips()
```

Comment:
Would be nice to extend the test to also try IPv6.

Reply:
We have Tests for IPv6 and it is expected to throw as in general IPv6 does not fragment and forces PMTU discovery. 

https://github.com/dotnet/runtime/blob/2dc8c956f98389418400dda5bda663d67cbea938/src/libraries/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs#L123

 #76410 is asking to change that but that is another works cross the platforms.  (if we decide to move forward)